pkg/helm/kube.go

```diff
@@ -34,27 +34,31 @@ var kubeClient *kube
 var once sync.Once
 
 // GetConfigInstance returns a Kubernetes interface based on the current configuration
-func getConfigInstance() *kube {
+func getConfigInstance(context string) *kube {
 	once.Do(func() {
 		if kubeClient == nil {
 			kubeClient = &kube{
-				Client: getKubeClient(),
+				Client: getKubeClient(context),
 			}
 		}
 	})
 	return kubeClient
 }
 
-func getKubeClient() kubernetes.Interface {
-	kubeConf, err := config.GetConfig()
+func getKubeClient(context string) kubernetes.Interface {
+	var clientset *kubernetes.Clientset
+
+	kubeConf, err := config.GetConfigWithContext(context)
```

Comment:
I assume `""` just falls back to `GetConfig`?

Reply:
Sort of? It seems to just use whatever `current-context` is.  I will add to the example output to demonstrate this.